### ISSUE TITLE
Cannot creat/edit files in C:/Programs Files/WindowsApps

### ISSUE THREAD

**Issue by Shomnipotence** [DavidXanatos/IgnoreACLs#2]
Cannot creat/edit files in C:/Programs Files/WindowsApps

**Comment by DavidXanatos** [comment]
could be in that case copy them someware else edit and copy them back where thay were from.

**Comment by Shomnipotence** [comment]
> could be in that case copy them someware else edit and copy them back where thay were from.

No, you can copy they from C:/Program Files/WindowsApps to other folder but you cannot  plaste edited files  to overwrite  the original file in the WindowsApps

**Comment by Shomnipotence** [comment]
> could be in that case copy them someware else edit and copy them back where thay were from.

Ah would you like to try it in Windows 11? plaste any file to the WindowsApps? I have failed

**Comment by DavidXanatos** [comment]
yea I'll test that on win 11

**Comment by Shomnipotence** [comment]
> yea I'll test that on win 11

but can it work on Windows 10？I mean plasting files to C:\Program Files\WindowsApps

**Comment by DavidXanatos** [comment]
for what i remember moving files worked fine only eee.. i think renaming was problematic as it uses a different call then copy move opearations

**Comment by DavidXanatos** [comment]
hmm... so i can edit delete and move files out of thet location on win 11,
but i fail to create new files or copy/move files there

**Comment by Shomnipotence** [comment]
> hmm... so i can edit delete and move files out of thet location on win 11, but i fail to create new files or copy/move files there

yeah, that is what I mean. So will you find a way to crack that? I think , my disk shold control by myself

**Comment by DavidXanatos** [comment]
> my disk shold control by myself
> 

Yes this is my opinion to I'll look into this if i can find a way to work around that limitation

**Comment by lili-cmd** [comment]
> Cannot creat/edit files in C:/Programs Files/WindowsApps

There is a bug on Windows 10,You are able to move the app to fat32 or exfat volumes，Although you can't double-click WindowsApps in this directory to access it, the app has lost its limits, and you can modify the app's files at will.But this is no longer available on win11

**Comment by lili-cmd** [comment]
> 在这种情况下，可以复制它们，否则请编辑并复制它们回到thay的来源。

This is also not available on win11 when this one.After the modification occurs the application will be unavailable, and if you access the Internet Windows Store will recover all files

**Comment by Mr-Mau** [comment]
I couldn’t manage to install the driver. I did put windows in test mode, right click on .inf file them install, but doesn’t work. I’ve tried in CMD with command: “rundll32 syssetup,SetupInfObjectInstallAction DefaultInstall 128 IgnoreACLs.inf” inside the file folder, but also returns with error.
In the readme say’s: “Install using the ini file” but there is no .ini file. Any help?
